examples/README.md

```diff
@@ -61,4 +61,5 @@ API Examples
 
 ### [Test Monitor](testmonitor)
 
-- [Results](testmonitor/results): Demonstrates how to use the SystemLink Test Monitor API to publish test results to the server.
\ No newline at end of file
+- [Results](testmonitor/results): Demonstrates how to use the SystemLink Test Monitor API to publish test results to the server.
+- [File Attachment](testmonitor/fileattachment): Demonstrates how to use the SystemLink Test Monitor API to attach files that are uploaded to the SystemLink server using the SystemLink File API.
```

Comment:
It's also fine to use hyphens if you think `file-attachment` would be better than `fileattachment`. I don't feel strongly either way.

Reply:
Renamed to file-attachment.